lib/pmdtester/cmd.rb

```diff
@@ -6,6 +6,17 @@ module PmdTester
   # Containing the common method for executing shell command
   class Cmd
     extend PmdTester
+    def self.stdout_of(cmd)
+      stdout, _stderr, _status = execute(cmd)
+      stdout&.chomp!
+      stdout
+    end
+
+    def self.stderr_of(cmd)
+      _stdout, stderr, _status = execute(cmd)
+      stderr
+    end
+
     def self.execute(cmd)
```

Comment:
How about changing the behavior of execute to return a combined string with stderr+stdout? That way, we wouldn't need to change the calls to execute everywhere....

You could keep your adjusted execute method and make it private and call it e.g. "internal_execute". And then provide the public method "execute" again with slightly adjusted behavior